archivist/notebooks/Sharing Artist Asset with Record Labels.ipynb

```diff
@@ -121,6 +128,12 @@
    "source": [
     "def create_access(arch):\n",
     "    \"\"\"\n",
+    "    Pre-requisite: Ensure that RKVST Subject IDS have been exchanged with parties that one\n",
+    "    wants to share with BEFORE creating this Access Policy.\n",
+    "\n",
+    "    How to exchange RKVST Subject IDs:\n",
+    "    https://docs.rkvst.com/docs/rkvst-basics/sharing-assets-with-obac/\n",
+    "\n",
```

Comment:
FYI In the notebook 'share Asset' is code that does a key exchange using 'arch.subjects.share'

Reply:
I like having both routes available in the examples - let's go with this one here